scripts/scripts.js

```diff
@@ -52,7 +52,9 @@ export function decorateMain(main) {
   // hopefully forward compatible button decoration
   decorateButtons(main);
   decorateIcons(main);
-  buildAutoBlocks(main);
+  if (!main.querySelector('.carousel')) buildAutoBlocks(main);
```

Comment:
I have added this condition to buildAutoBlocks rather  - 

function buildAutoBlocks(main) {
  try {
    if (!main.querySelector('.carousel')) buildHeroBlock(main);
    else if (!main.querySelector('.carousel').contains(main.querySelector('h1'))) buildHeroBlock(main);
    // buildHeroBlock(main);
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error('Auto Blocking failed', error);
  }
}

Since it will check this condition first before even hitting buildHeroBlock function. 

https://hero-carousel-17--ororafresh--hlxsites.hlx.page/Drafts/Meet/produce-packaging  - where Hero autoblocking is there, is enabled. 

Please advise.